website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -1353,6 +1353,20 @@ cluster-name = 'cluster'
 'memory.available' = '12%%'
 ```
 
+#### Device ownership in Bottlerocket
```

Comment:
We need to make this update main. If we are intending to update the docs

Reply:
Yup - it's there 👍🏻 https://github.com/aws/karpenter-provider-aws/blob/main/website/content/en/preview/concepts/nodeclasses.md#device-ownership-in-bottlerocket